app/services/add_remote_tags.rb

```diff
@@ -1,6 +1,6 @@
 class AddRemoteTags < RemoteTaggingService
-  def process(tag)
-    post_request(object_url, [tag])
+  def process(tags)
+    post_request(object_url, tags)
```

Comment:
@mkanoor You could also do this as `Array(tags)` which would leave an array alone and wrap a non-array element in an array.  Then the caller does not have to always pass an array for a single element.  Same with `DeleteRemoteTags`.

Reply:
@gmcculloug Then it doesn't match the class name, we would have to call the class to indicate that it can take a scalar or an array argument

Reply:
Not sure I fully agree about having to rename the class and is weird to force the caller to artificially create an array to pass to the method.  I'll try to find some good examples, but we can deal with that at a later time.  

I would rather move forward with the fix and discuss later.